### DISCUSSION TITLE
Storage clear

### DISCUSSION THREAD

**Discussion by LuridG** [anyproto/contributors#90] [Q&A]
I have been using Anytype because of its impeccable local file mechanism. However, I've encountered some issues and would like to know how to resolve them. After successfully importing a 2.2GB file from Notion using the API, I switched the software version to the beta (or test) version. Subsequently, I noticed that the local storage had been cleared, and everything was transferred to a 1.07GB cloud storage. Given that my original file was 2.2GB, I'm uncertain about how to retrieve the missing content. Could this issue be attributed to my switch from the official version to the beta version? Will a regular upgrade in the official version also result in the loss of local files? Additionally, is there an option to customize the location of local file storage? This would facilitate easier backups.

**Comment by isle9** [comment]
Is this one singular file? Are you able to open it, and is everything still functional?
It's probably not related to you switching to the pre-relase version (if this is what you switched to), but I guess it could be connected.
Per our roadmap, custom storage location is [coming soon](https://github.com/orgs/anyproto/projects/1?pane=issue&itemId=29227794), but the best way to backup your data is to export it using the Any-block format.

**Reply by LuridG** [reply]
It's not just a singular file, but an entire space that was 2.2GB in size. Before switching to the beta version, I was able to open every notes that contained images. However, after switching to the pre-release version, some of the notes, the images seem to be missing, though the text remains intact. I greatly appreciate your suggestion about exporting in the Any-block format.

**Reply by isle9** [reply]
The best thing for you to do would be to post this in our community forum so that our devs can help you directly. Also, after posting a bug report, you could try to create a new account and import your stuff from Notion after you've already switched to the pre-release version to check if that was the problem.
